x/ccv/provider/client/cli/tx.go

```diff
@@ -350,6 +352,12 @@ Example:
     "allow_inactive_vals": false
    }
 }
+
+Note that only 'consumer_id' is mandatory. The others are optional.
+Not providing one of them will leave the existing values unchanged. 
+Providing one of 'metadata', 'initialization_parameters' or 'power_shaping_parameters', 
+will update all the containing fields. 
+If one of the fields is missing, it will be set to its zero value.
```

Comment:
Later when you do `NewMsgUpdateConsumer(signer, ...)`, can we rename `signer` to `owner` internally? The same way you did the renaming of `signer` to `submitter` for `MsgCreateConsumer`?